### PR TITLE
[Flutter Parent][MBL-13655] View message attachments

### PR DESCRIPTION
**Pull request by JordanMarshall**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019310&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/utils/common_widgets/view_attachment/viewers/text_attachment_viewer.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  var _referenceTextSize;
+
+  @override
+  Widget build(BuildContext context) {
+    return AttachmentFetcher(
+      attachment: widget.attachment,
+      builder: (context, file) {
+        return LayoutBuilder(
+          builder: (context, size) => GestureDetector(
+            onScaleStart: (_) => _referenceTextSize = _textSize,
```
Comment (Blackchai) [inline apps/flutter_parent/lib/utils/common_widgets/view_attachment/viewers/text_attachment_viewer.dart]: noice

**Comment by JordanMarshall** [comment]
That's the loading indicator shown by PhotoView. There might be a way to override that but I'd have to look into it.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019613&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019614&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**